backend/src/services/group/group.service.ts

```diff
@@ -66,12 +66,24 @@ class Group {
     };
   }
 
-  public async getPaginated({
-    page,
-    count,
-  }: EntityPaginationRequestQueryDto): Promise<
-    EntityPagination<GroupsItemResponseDto>
-  > {
+  public async getAll(
+    paginationData?: EntityPaginationRequestQueryDto,
+  ): Promise<EntityPagination<GroupsItemResponseDto>> {
+    const { page, count } = paginationData ?? {};
+
+    if (!page || !count) {
```

Comment:
add default values

repository's method should be also `getAll` with pagination options

Reply:
@what1s1ove I didn't quite understand, should we have methods, where we get all values, without pagination? Or we only get paginated data?